src/Nethermind/Nethermind.BlockValidation/IFlashbotsRpcModule.cs

```diff
@@ -0,0 +1,19 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System.Threading.Tasks;
+using Nethermind.BlockValidation.Data;
+using Nethermind.JsonRpc;
+using Nethermind.JsonRpc.Modules;
+
+namespace Nethermind.BlockValidation;
+
+[RpcModule(ModuleType.Flashbots)]
+public interface IFlashbotsRpcModule : IRpcModule
+{
+    [JsonRpcMethod(
+        Description = " validate the builder submissions as received by a relay",
+        IsSharable = true,
```

Comment:
definitely not sharable, it needs to be instance based as each instance will have it's own vision of what the state is.